src/index.js

```diff
@@ -6,10 +6,12 @@ import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
 import {
+  HistoryProvider,
   LikeProvider,
   ToastProvider,
   UserProvider,
```

Comment:
from your context file, how are you importing `UserProvider`. I tried searching for the file, but I think it is not present.

Reply:
PR shows only the changed files. Maybe that's the reason file is not being shown here !

Reply:
Yeah! can be.